packages/core/src/components/cms/RenderSections.tsx

```diff
@@ -55,15 +53,6 @@ export const LazyLoadingSection = ({
   sectionName: string
   children: ReactNode
 }) => {
-  const { cart: displayCart, modal: displayModal } = useUI()
-
-  if (SECTIONS_OUT_OF_VIEWPORT.includes(sectionName)) {
```

Comment:
suggestion: I believe SECTIONS_OUT_OF_VIEWPORT could be deleted now

Reply:
as we talked we decided on keep the SECTIONS_OUT_OF_VIEWPORT for the sections that should load without the ViewportObserver :)